docs/en_us/internal/testing.rst

```diff
@@ -610,6 +616,24 @@ During acceptance test execution, Django log files are written to
 
 **Note**: The acceptance tests can *not* currently run in parallel.
 
+
+Testing internationalization with dummy translations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any text you add to the platform should be internationalized. To generate
+translations for your new strings, run the following:
```

Comment:
The heading does have "dummy translations" but possibly make it doubly clear in the paragraph: "To generate dummy translations for your new strings..."